session_0/README.md

```diff
@@ -17,6 +17,8 @@ Rails. You can refer to books on programming or websites like
 
 > Read the [installation guide](/installation.md) and [setting up local
 > workspace](/essential_git.md) to get started.
+>
```

Comment:
Please don't upload Session 0 and Session 1 related files - using branches and pushing code correctly is part of learning how to use Git.

Reply:
What I did was :
git add .
git commit

So, I'm not sure why only session_1 files got added not session_0. Like, either both shd be uploaded or none of them uploaded.
Also, for upcoming sessions, should I do only 'git add session_id' to make that only session to be added in the branch.

Reply:
Yes

Reply:
ohk..Gr8 :+1: